packages/web-app/src/routes/questionnaires-for-review/check-and-confirm.spec.js

```diff
@@ -0,0 +1,17 @@
+const { get } = require('../../../test/routes/router-mock');
+const checkAndConfirmController = require('../../controllers/questionnaires-for-review/check-and-confirm');
+
+describe('routes/check-and-confirm', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line global-require
+    require('./check-and-confirm');
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should define the expected routes', () => {
+    expect(get).toHaveBeenCalledWith('/:appealId', checkAndConfirmController.getCheckAndConfirm);
```

Comment:
It looks like the path should be `/` as the route's not using the `appealId` param.